portfolio/src/main/webapp/script.js

```diff
@@ -12,41 +12,110 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let navbar;
+let navOffset;
+
+window.onload = function() {
+  navbar = document.getElementById('navbar');
+  navOffset = navbar.offsetTop;
+  getComments();
+};
+
+window.onscroll = function() {
+  makeNavSticky();
+};
+
+function makeNavSticky() {
+  if (window.pageYOffset >= navOffset) {
+    navbar.classList.add('sticky');
+  } else {
+    navbar.classList.remove('sticky');
+  }
+}
+
 /**
- * Adds a random greeting to the page.
+ * Adds a random Avatar line to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['That\'s rough buddy', 'My cabbages!', 'Flameo, hotman!', 'Who are you? And what do you want?'];
+function addRandomLine() {  // eslint-disable-line no-unused-vars
+  const lines = [
+    'That\'s rough buddy.',
+    'My cabbages!',
+    'Flameo, hotman!',
+    'Yip yip!',
+    `Instead of seeing what they want you to see, you got to open your brain
+     to the possibilities.`,
+    'Life happens wherever you are, whether you make it or not.',
+    'Boomerang! You do always come back!',
+    'It\'s a giant mushroom! Maybe it\'s friendly!',
+    'I am not Toph! I am Melon-Lord! Muah-ha-ha-ha-ha!',
+  ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random line.
+  const line = lines[Math.floor(Math.random() * lines.length)];
 
   // Add it to the page.
-  const container = document.getElementById('greeting-container');
-  container.innerText = greeting;
-  
+  const container = document.getElementById('avatar-container');
+  container.innerText = line;
 }
 
+function getComments() {  // eslint-disable-line no-unused-vars
```

Comment:
Curious - why is this needed? I don't see any unused vars here.

Reply:
I added this before getComments() was called elsewhere in the script. I'll make sure to delete!